client/src/components/layouts/global.css

```diff
@@ -53,11 +54,26 @@ body {
 }
 
 .default-layout {
-  flex: 1 0 auto;
   margin-top: var(--header-height, 0);
   background: var(--secondary-background);
 }
 
+.default-layout:only-child {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  min-height: 0;
```

Comment:
This one, I found its usage, as it shows the buttons always

![image](https://user-images.githubusercontent.com/88248797/201744070-cff8b59d-392d-4f88-bc9a-0f9f361c345e.png)


But couldn't find `flex: 1 1 auto` usage

Reply:
In [Basic JavaScript / Comment Your JavaScript Code](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comment-your-javascript-code), [Bootstrap / Create a Block Element Bootstrap Button](https://www.freecodecamp.org/learn/front-end-development-libraries/bootstrap/create-a-block-element-bootstrap-button), or [Learn HTML by Building a Cat Photo App / Step 69](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-69), the `flex: 1 1 auto` is used for stretching `#mobile-layout-pane-editor`, `#mobile-layout-pane-console`, `#mobile-layout-pane-preview` of mobile layout, and `.reflex-container.vertical` of desktop layout in Steps to fill remaining space vertically.